### PR TITLE
Issue #SB-18941 chore: Update the mobile app setup documentation

### PR DESCRIPTION
**Pull request by AmiableAnil**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @AmiableAnil,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://583.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://583.qa.docs.sunbird.org/2.5.0/developer-docs/mobile-app-installation// <br />  <br/> **Modified sections are -** <br />

File: developer-docs/mobile-app-installation/index.md [hunk 1]
```diff
@@ -48,10 +48,13 @@ Replace the producer id `PRODUCER_ID` for respective environments in sunbird.pro
...
 
-## Build APK from Git Repository
-Sunbird mobile app can be built from the main source code which is available at [Sunbird-Ed](https://github.com/Sunbird-Ed)/[SunbirdEd-mobile](https://github.com/Sunbird-Ed/SunbirdEd-mobile).
+## Setup mobile app workspace from Git Repository
+Sunbird mobile app can be built from the main source code which is available at  [SunbirdEd-mobile-app](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app).
 
-Sample properties file is located inside buildConfig folder. This has to be renamed to sunbird.properties and appropriate values should be provided.
+* Clone the [SunbirdEd-mobile-app](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app) repo
+* Customising App Configuration - sample.sunbird.properties file is located inside buildConfig folder. This has to be renamed to sunbird.properties and appropriate values should be provided.
+* Package the framework and form data
```
Comment (NupurSancheti) [inline developer-docs/mobile-app-installation/index.md:56]: Please explain how to package

File: developer-docs/mobile-app-installation/index.md [hunk 2]
```diff
@@ -48,10 +48,13 @@ Replace the producer id `PRODUCER_ID` for respective environments in sunbird.pro
...
+* go to project folder and run **./build.sh**
```
Comment (NupurSancheti) [inline developer-docs/mobile-app-installation/index.md:57]: Through command line?

File: developer-docs/mobile-app-installation/index.md [hunk 3]
```diff
@@ -48,10 +48,13 @@ Replace the producer id `PRODUCER_ID` for respective environments in sunbird.pro
...
 Replace `release_fabric_api_key` in `sunbird.properties` with your fabric API Key. Create an account in [fabric.io](https://get.fabric.io/) and register in the app to get the API key.
 
-## Build APK from Git Repository
-Sunbird mobile app can be built from the main source code which is available at [Sunbird-Ed](https://github.com/Sunbird-Ed)/[SunbirdEd-mobile](https://github.com/Sunbird-Ed/SunbirdEd-mobile).
+## Setup mobile app workspace from Git Repository
+Sunbird mobile app can be built from the main source code which is available at  [SunbirdEd-mobile-app](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app).
 
-Sample properties file is located inside buildConfig folder. This has to be renamed to sunbird.properties and appropriate values should be provided.
+* Clone the [SunbirdEd-mobile-app](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app) repo
+* Customising App Configuration - sample.sunbird.properties file is located inside buildConfig folder. This has to be renamed to sunbird.properties and appropriate values should be provided.
```
Comment (NupurSancheti) [inline developer-docs/mobile-app-installation/index.md:55]: Is this step to be performed on explorer or command line env?

File: developer-docs/mobile-app-installation/index.md [hunk 4]
```diff
@@ -13,10 +13,10 @@ The Sunbird Mobile app provides mobility to its feature-rich learning platform.
...
 <br>1.To set up Sunbird mobile app, ensure you have installed the following:
-<br>&emsp;a) NPM Version - 3.8.+
-<br>&emsp;b) Node JS Version -  6.+
-<br>&emsp;c) Cordova Version - 8.0.0
-<br>&emsp;d) Ionic Version - 3.20.0
+<br>&emsp;a) NPM Version - 6.+
+<br>&emsp;b) Node JS Version -  8.+
+<br>&emsp;c) Cordova Version - 8.+
+<br>&emsp;d) Ionic Version - 4.11.2
 <br>2. Generate the key and secret for the mobile app user using the JWT token of the mobile admin user. Run the **OnboardConsumers** Jenkins Job and take the jwt token(**JWT token for mobile_admin**) from Jenkins Output.
```
Comment (NupurSancheti) [inline developer-docs/mobile-app-installation/index.md:20]: this seems to be incomplete

**Comment by keshavprasadms** [comment]
Hello @AmiableAnil,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://583.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://583.qa.docs.sunbird.org/2.5.0/developer-docs/mobile-app-installation// <br />  <br/> **Modified sections are -** <br />

**Comment by NupurSancheti** [comment]
@Sunitashelar: Please merge the PR

**Pull request merged**